doc/getting_started/installation_linux.rst

```diff
@@ -210,87 +210,89 @@ environment is recommended to avoid interfering with your system Python.
 Install the Zephyr Software Development Kit (SDK)
 *************************************************
 
-Use of the Zephyr SDK is optional, but recommended. Some of the dependencies
-installed above are only needed for installing the SDK.
+Use of the Zephyr SDK is highly recommended and may even be required under
+certain conditions (e.g. running tests in QEMU for some architectures). Some
+of the dependencies installed above are only needed for installing the SDK.
```

Comment:
which ones? `wget` and `unzip`? Why the need to mention this?

Reply:
I kept that sentence because it was already there; but, rethinking about it, it would be better to just remove it to avoid confusions.

Reply:
done